inference_perf/reportgen/base.py

```diff
@@ -256,10 +249,6 @@ def generate_prometheus_metrics_report(
                         name=f"stage_{stage_id}_prometheus_metrics",
                         contents=summarize_prometheus_metrics(collected_metrics).model_dump(),
                     )
-                    if report_file is not None:
-                        logger.info(
```

Comment:
These log lines were useful to let people know what files are generated and where. Otherwise after the run, they might be looking around for the reports. So, if we can keep at least one of these in the output pointing to the report directory or files, that would be good.